src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/theme/css.txt

```diff
@@ -30,4 +30,4 @@ components/searchresults/searchresults-component.css
 
 components/productteaser/teaser.css
 
-components/teaser/teaser.css
+components/teaser/teaser.css
```

Comment:
Just to be 100% sure, this is probably another issue you spotted when working on this?

Reply:
I just merged some changes from `master` and this happend 🤷